src/mamba/py_interface.cpp

```diff
@@ -20,6 +20,8 @@
 #include "mamba/core/transaction.hpp"
 #include "mamba/core/url.hpp"
 #include "mamba/core/util.hpp"
+#include "mamba/core/virtual_packages.hpp"
+#include "../core/virtual_packages.cpp"
```

Comment:
I think you need to remove that include and add the translation unit in the setup.py if you want to use it from mamba https://github.com/mamba-org/mamba/blob/fe4ecc5061a49c5b400fa7e7390b679e983e8456/setup.py#L121